base/src/main/java/proguard/optimize/inline/InstructionAtOffset.java

```diff
@@ -0,0 +1,45 @@
+package proguard.optimize.inline;
+
+import proguard.classfile.instruction.Instruction;
+
+import java.util.Objects;
+
+public final class InstructionAtOffset {
```

Comment:
Is it not more safe to use the CodeAttribute and Instruction for this? Using existing data types, classes,... helps to make use of the existing infrastructure which makes a lot of stuff easier. The infrastructure helps by performing updates on the data structures and performing validity checks.

Reply:
The problem is that if we only store a CodeAttribute and an Instruction we can't get the correct offset of the instruction 
reliably.